qa/update-tests/src/test/java/io/camunda/zeebe/test/ContainerState.java

```diff
@@ -78,14 +77,20 @@ ZeebeClient client() {
 
   ContainerState withOldBroker() {
     broker(PREVIOUS_VERSION);
-    // user needs to be set to allow a smooth update from zeebe 8.2 to 8.3
-    // as the default user changed to `zeebe` with 8.3 and was `root` with 8.2
-    // TODO remove after 8.3 release
-    withUser(ZEEBE_USER);
+    // user - needs to be set to `1001` to allow a smooth update from zeebe 8.3 to 8.4,
+    // as the default user changed to `1001` with 8.4 and was `1000` with 8.3
+    // TODO remove after 8.4 release
+    withUser("1001");
     return this;
   }
 
   ContainerState withNewBroker() {
+    // user - `1001` is the default in 8.4
+    // group - needs to be set to `0` as the data volume in 8.3 is owned by 1000:0
+    // thus zeebe 8.4 needs to run with group `0` to be able to create new files in
+    // the root of the data volume (in particular it creates a new `.topology.meta` file)
+    // TODO remove after 8.4 release
+    withUser("1001:0");
```

Comment:
this is unfortunately needed, as the docker data volume keeps the ownership set to `1000:0` as set here https://github.com/camunda/zeebe/blob/stable/8.3/Dockerfile#L161-L162 which lead to 8.4 failing to create the `.topology.meta` file :/ 

On k8s this shouldn't be an issue though as the permissions should get updated via fsGroup, thus we only need this hack in the tests, curious if you have a better idea @npepinpe .